mysite/pages/models.py

```diff
@@ -21,7 +21,7 @@ class Task(models.Model):
     complete = models.BooleanField(default=False)
     todolist = models.ForeignKey(ToDoList, on_delete=models.CASCADE)
     description_text = models.CharField(default="Untitled Task", max_length=200)
-    due_date = models.DateTimeField('Due Date', default=timezone.now())
+    due_date = models.DateTimeField('Due Date', default=timezone.now)
```

Comment:
Minor change here, if we try to call a function like this the due_date will be statically set to the time the task class is actually defined - so I changed it to pass the actual function (rather than the function's result).